<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>851</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../../../Desktop/abstract-photography-wallpaper-2.jpg</pixmap>
    </property>
   </widget>
   <widget class="QListWidget" name="adminList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>611</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="photoLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>640</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Photo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="photoLineEdit">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>640</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lengthLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>672</y>
      <width>58</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Length</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lengthLineEdit">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>672</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sizeLineEdit">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>512</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sizeLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>512</y>
      <width>37</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>544</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="colorLineEdit">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>544</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="priceLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>576</y>
      <width>45</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="priceLineEdit">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>576</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="quantityLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>608</y>
      <width>76</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="quantityLineEdit">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>608</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>650</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>UPDATE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>580</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>510</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QListWidget" name="userList">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>340</y>
      <width>231</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>90</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../../../Desktop/descÄƒrcare.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="filterButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>720</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FILTER</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="shuffleRadioButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>730</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string> SHUFFLE </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sortRadioButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>760</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SORT</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>16</width>
      <height>371</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>540</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BUY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>580</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CHECK PHOTO</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>620</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>         SAVE TO CSV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>         SAVE TO HTML</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>460</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Garamond</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>USER OPTIONS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>790</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>670</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OPEN CSV/HTML</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>userList</zorder>
   <zorder>photoLabel</zorder>
   <zorder>photoLineEdit</zorder>
   <zorder>lengthLabel</zorder>
   <zorder>lengthLineEdit</zorder>
   <zorder>sizeLineEdit</zorder>
   <zorder>sizeLabel</zorder>
   <zorder>colorLabel</zorder>
   <zorder>colorLineEdit</zorder>
   <zorder>priceLabel</zorder>
   <zorder>priceLineEdit</zorder>
   <zorder>quantityLabel</zorder>
   <zorder>quantityLineEdit</zorder>
   <zorder>updateButton</zorder>
   <zorder>deleteButton</zorder>
   <zorder>addButton</zorder>
   <zorder>adminList</zorder>
   <zorder>label_2</zorder>
   <zorder>filterButton</zorder>
   <zorder>shuffleRadioButton</zorder>
   <zorder>sortRadioButton</zorder>
   <zorder>verticalSlider</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>comboBox</zorder>
   <zorder>userLabel</zorder>
   <zorder>homeButton</zorder>
   <zorder>pushButton_3</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>colorLineEdit</tabstop>
  <tabstop>priceLineEdit</tabstop>
  <tabstop>quantityLineEdit</tabstop>
  <tabstop>photoLineEdit</tabstop>
  <tabstop>lengthLineEdit</tabstop>
  <tabstop>adminList</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>sizeLineEdit</tabstop>
  <tabstop>updateButton</tabstop>
  <tabstop>deleteButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
